docs/guides/search-path.md

```diff
@@ -87,9 +92,9 @@ Queries in Steampipe [Mods](https://steampipe.io/docs/mods/overview) are written
 By default, Steampipe sets the schema search path as follows:
 1. The `public` schema first.  This schema is writable, and allows you to create your own objects (views, tables, functions, etc).
 2. Connection schemas, in **alphabetical order** by default.
-3. The `internal` schema last.   This schema contains steampipe built-in functions and other internal Steampipe objects.  This schema is not displayed or managed by the steampipe search path commands and options -  it is always controlled by steampipe (though you will see it in native SQL commands, such as `show search_path`).
+3. The `internal` schema last.   This schema contains Steampipe built-in functions and other internal Steampipe objects.  This schema is not displayed or managed by the Steampipe search path commands and options. However you'll see it in native SQL commands such as `show search_path`.
```

Comment:
In this case, I believe `However` requires a comma afterward since you are using it as a connector.

Reply:
How about: "This schema is not displayed or managed by the Steampipe search path commands and options, but you'll see it in native SQL commands such as `show search_path`."

Reply:
wfm